js/main.js

```diff
@@ -25,7 +18,80 @@ function submitForm(event) {
   data.nextEntryId++;
   data.entries.unshift(entry);
   changeImage.setAttribute('src', 'images/placeholder-image-square.jpg');
-
+  var generatedEntry = createEntry(entry);
+  $list.prepend(generatedEntry);
+  viewEntries();
   $form.reset();
 
 }
+
+function createEntry(entry) {
+  var $entry = document.createElement('li');
+  var $row = document.createElement('div');
+  var $columnOne = document.createElement('div');
+  var $img = document.createElement('img');
+  var $columnTwo = document.createElement('div');
+  var $title = document.createElement('h2');
+  var $notes = document.createElement('p');
+
+  $entry.className = 'entry';
+  $row.className = 'row';
+  $columnOne.className = 'column-half';
+  $img.className = 'empty-photo';
+  $img.setAttribute('src', entry.urlPhoto);
+  $img.setAttribute('alt', 'entry image');
+  $columnTwo.className = 'column-half';
+  $title.className = 'entry-title';
+  $title.textContent = entry.title;
+  $notes.className = 'entry-notes';
+  $notes.textContent = entry.notes;
+
+  $entry.appendChild($row);
+  $row.appendChild($columnOne);
+  $columnOne.appendChild($img);
+  $row.appendChild($columnTwo);
+  $columnTwo.appendChild($title);
+  $columnTwo.appendChild($notes);
+
+  return $entry;
+}
+
+function DOMContentLoaded(event) {
```

Comment:
I would five this function a different name, and one with a lowercase first letter.